<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_data</class>
 <widget class="QDialog" name="dialog_data">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>340</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>410</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>410</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1a1a1a;
color: #ffffff;</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffbe00;
font: 23px &quot;Lato&quot;;</string>
    </property>
    <property name="text">
     <string>SETTING</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>410</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#242424;
font: 15px &quot;Lato Black&quot;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="apply_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:#2d2d2d;
border:none;</string>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>410</width>
     <height>240</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
	background:transparent;
	border: 3px solid #000000;
}
QPushButton {
    border: 2px solid #5e5e5e;
	color: #5e5e5e;
    background-color: transparent;
	font: 15px &quot;Lato Black&quot;
}
QPushButton:checked {
	border-color: #ffbe00;
 	color: #ffbe00;
    background-color: rgba(0, 0, 0, 87);
}
QRadioButton{
	background: none;
	color: #7f7f7f;
	font: bold 15px &quot;Lato&quot;;
}
QRadioButton:Checked{
	color: #ffbe00;
}
QLabel{
	background:transparent;
	border: none;
	color: #dddddd;
	font: 18px &quot;Lato&quot;;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Database Version</string>
    </property>
   </widget>
   <widget class="QLabel" name="current_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15px;</string>
    </property>
    <property name="text">
     <string>Current:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastest_label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15px</string>
    </property>
    <property name="text">
     <string>Lastest:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>410</width>
      <height>90</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	border:none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DLC Packs</string>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_gf">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GF</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_chu">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CHU</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_tr">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TR</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_bs">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>BS</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_p3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>P3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_ve">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>VE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_gc">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>GC</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_t1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>T1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_t2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>T2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_t3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>T3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_dm">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>DM</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_ce">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_es">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ES</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="yd_cb_cy">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CY</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Title (A to Z)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>WIP</string>
    </property>
   </widget>
  </widget>
  <zorder>frame_3</zorder>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
